packages/engine/tests/units/state/nullable_fixed_size_list/src/init-js.json

```diff
@@ -0,0 +1,12 @@
+[
+  {
+    "behaviors": ["empty.js"]
+  },
```

Comment:
Making this a thread

> Sadly, null values are not propagated to json_state.json, so we currently can only check, if the second agent has the right values.

Aside from having to change the testing logic, there isn't anything stopping us from checking the _absence_ of fields. (Which might be a good addition anyway, if we explicitly put `null` in the expected output and verify it's either missing or `null` in the state.json)

But in the mean-time can't we do something similar to the other tests, where instead of an empty behavior we can read the variable and set it on a new one in state?

Reply:
> But in the mean-time can't we do something similar to the other tests, where instead of an empty behavior we can read the variable and set it on a new one in state?

Good idea, will implement this.